stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/decorator/internal/AvatarDecorator.kt

```diff
@@ -44,6 +46,20 @@ internal class AvatarDecorator(
         setupAvatar(getAvatarView(viewHolder.binding.avatarMineView, viewHolder.binding.avatarView, data.isMine), data)
     }
 
+    /**
+     * Decorates the avatar of the link attachment message, based on the message owner.
+     *
+     * @param viewHolder The holder to decorate.
+     * @param data The item that holds all the information.
+     */
+    override fun decorateLinkAttachmentsMessage(
+        viewHolder: LinkAttachmentsViewHolder,
+        data: MessageListItem.MessageItem,
+    ) {
+        controlVisibility(viewHolder.binding.avatarMineView, viewHolder.binding.avatarView, data.isMine)
+        setupAvatar(getAvatarView(viewHolder.binding.avatarMineView, viewHolder.binding.avatarView, data.isMine), data)
```

Comment:
Do we need text decoration and/or markdown stuff here?

Reply:
Was this possibly supposed to be left in `TextDecorator`?

Anyways, `LinkAttachmentsViewHolder` has it's own `ChatMessageTextTransformer`. This is the aproach we have usually been taking with text transformations.